apps/cli/templates/defaults/template.tsx

```diff
@@ -0,0 +1,140 @@
+// This file was generated by pnpm prebuild
```

Comment:
Is it generated? Shouldn't __generated__/template be used for this?

Reply:
Actually not generated, it's a template file. That line should be removed 👍